Sources/SwiftProtobuf/FieldTag.swift

```diff
@@ -0,0 +1,65 @@
+// SwiftProtobuf/Sources/SwiftProtobuf/FieldTag.swift - Describes a binary field tag
```

Comment:
Since FieldTag and WireFormat are specific to protobuf binary coding, I would prefer the source files be named accordingly.